### PR TITLE
rls changes

### PR DESCRIPTION
**Pull request by saltcod**
Consolidates the two RLS UIs, removing the feature preview

screenshot-2024-09-18-at-16 34 02

### CODE REVIEW

File: apps/studio/components/interfaces/Auth/Policies/AIPolicyEditorPanel/index.tsx [hunk 1]
```diff
@@ -122,8 +125,10 @@ export const AIPolicyEditorPanel = memo(function ({
 
   // Use chat id because useChat doesn't have a reset function to clear all messages
   const [chatId, setChatId] = useState(uuidv4())
-  const [tabId, setTabId] = useState<'templates' | 'conversation'>('templates')
-
+  const [tabId, setTabId] = useState<'templates' | 'conversation'>(
+    editView as 'templates' | 'conversation'
+  )
+  console.log({ editView }, { tabId })
```
Comment (jordienr) [inline apps/studio/components/interfaces/Auth/Policies/AIPolicyEditorPanel/index.tsx:131]: police get him

File: apps/studio/components/interfaces/Auth/Policies/AIPolicyEditorPanel/index.tsx [hunk 2]
```diff
@@ -180,6 +187,29 @@ export const AIPolicyEditorPanel = memo(function ({
...
+      return (
+        definition
+          .trim()
+          .replace(/\s*;\s*$/, '')
+          .replace(/\n\s*$/, '') + ';'
+      )
+    })
+    .join('\n\n')
+
+  console.log({ existingPolicyDefinition })
```
Comment (jordienr) [inline apps/studio/components/interfaces/Auth/Policies/AIPolicyEditorPanel/index.tsx:212]: police

File: apps/studio/components/interfaces/Auth/Policies/PolicyTableRow/PolicyTableRowHeader.tsx [hunk 3]
```diff
@@ -70,21 +76,41 @@ const PolicyTableRowHeader = ({
...
+                    setEditView('conversation')
+                  }}
+                >
+                  <AiIconAnimation className="scale-75 [&>div>div]:border-black dark:[&>div>div]:border-white" />
+                </Button>
+              </TooltipTrigger_Shadcn_>
+              <TooltipContent_Shadcn_ side="top">
+                {!canToggleRLS
+                  ? 'You need additional permissions to create RLS policies'
+                  : 'Create with the Supabase Assistant'}
```
Comment (jordienr) [inline apps/studio/components/interfaces/Auth/Policies/PolicyTableRow/PolicyTableRowHeader.tsx:111]: "Create (or generate?) with Supabase Assistant" sounds more natural to me here but i may be wrong

**Review by jordienr — APPROVED** [review]
This would be a really nice to have, but could we change the prompt if the user doesn't have any policies, so when you click "suggest" it will just suggest them instead of asking for the existing policies? ping me in slack if it's not clear what I'm trying to say.

CleanShot 2024-09-19 at 12 14 03@2x

File: apps/studio/components/interfaces/Auth/Policies/PolicyTableRow/PolicyTableRowHeader.tsx (hunk 3 above)
Comment (saltcod) [reply apps/studio/components/interfaces/Auth/Policies/PolicyTableRow/PolicyTableRowHeader.tsx:111]: yup sounds good

**Comment by saltcod** [comment]
entities aren't loading is why it's saying "no idea about this table" 
screenshot-2024-09-19-at-14 05 22

**Comment by saltcod** [comment]
checked staging and looks good

**Pull request merged**